x-pack/metricbeat/module/mssql/_meta/kibana/7/dashboard/Metricbeat-mssql-performance.json

```diff
@@ -0,0 +1,728 @@
+{
```

Comment:
I see you put this under `7`. I would assume automation around this does not work yet?

Reply:
I just tested this and it seems to collect the dashboard but not convert the JSON format. I would recommend it for now to put it under the 6 directory and then figure out what we do with it as soon as we resolved https://github.com/elastic/beats/issues/10069